requirements_test.txt

```
@@ -5,3 +5,5 @@ coverage~=5.5
 pre-commit~=2.13
 pytest-cov~=2.11
 tbump~=6.3.2
+types-typed-ast
```

```suggestion
types-typed-ast; implementation_name=="cpython" and python_version<"3.8"
```
Must sure if it's necessary as this is just a types package. Shouldn't harm though